client/packages/system/src/Encounter/ListView/AdditionalInformationColumn.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { Typography } from '@common/components';
+import {
+  ColumnDataAccessor,
+  ColumnDefinition,
+  ColumnDescription,
+  RecordWithId,
+} from '@openmsupply-client/common';
+
+export const getAdditionalInformationColumn = <T extends RecordWithId>(
+  accessor: ColumnDataAccessor<T>
+): ColumnDefinition<T> | ColumnDescription<T> => ({
+  label: 'label.additional-info',
+  key: 'events',
+  sortable: false,
+  Cell: ({ rowData }) => {
+    const additionalInfo = accessor({ rowData }) as string[];
+
+    if (!additionalInfo) return null;
```

Comment:
Unfortunately there isn't a type parameter to specify the ColumnDataAccessor return type, but do we expect that the accessor returns undefined?

Reply:
If it's empty (length === 0), this will return false.

Reply:
[f4f4c62](https://github.com/openmsupply/open-msupply/pull/1940/commits/f4f4c62dc815f9aa96d29723cce17a4957507311)